python/pyarrow/_compute.pyx

```diff
@@ -2251,3 +2339,208 @@ cdef CExpression _bind(Expression filter, Schema schema) except *:
 
     return GetResultValue(filter.unwrap().Bind(
         deref(pyarrow_unwrap_schema(schema).get())))
+
+
+cdef class ScalarUdfContext:
+    """
+    A container to hold user-defined-function related
```

Comment:
Docstrings should follow the proper numpydoc format (https://numpydoc.readthedocs.io/en/latest/format.html, also see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html)

Reply:
👍